tests/Propel/Tests/Runtime/ActiveQuery/ModelCriteriaGroupByArrayTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license MIT License
+ */
+
+namespace Propel\Tests\Runtime\ActiveQuery;
+
+use Propel\Tests\Helpers\Bookstore\BookstoreEmptyTestBase;
+use Propel\Tests\Bookstore\Book;
+use Propel\Tests\Bookstore\Author;
+use Propel\Tests\Bookstore\AuthorQuery;
+
+class ModelCriteriaGroupByArrayTest extends BookstoreEmptyTestBase
```

Comment:
you're having in this class database access, so you need to mark this class as "@database".

```php
/**
 * @group database
 */
class ModelCriteriaGroupByArrayTest extends BookstoreEmptyTestBase
```